tests/unit/io_queue_test.cc

```diff
@@ -510,3 +510,36 @@ SEASTAR_TEST_CASE(test_request_iovec_split) {
 
     return make_ready_future<>();
 }
+
+SEASTAR_THREAD_TEST_CASE(test_tb_params) {
+    disk_config_params disk_config(1);
+
+    disk_params d;
+    d.read_bytes_rate = 16 << 30;  // 16GB/s
+    d.write_bytes_rate = 16 << 30; // 16GB/s
+    d.read_req_rate = 150000; // 150k req/s
+    d.write_req_rate = 150000; // 150k req/s
+
+    auto io_config = disk_config.generate_config(d, 0, 1);
+
+    io_queue_for_tests tio(io_config);
+
+    auto cost_read_512 = tio.queue.request_capacity(internal::io_direction_and_length(internal::io_direction_and_length::read_idx, 512));
```

Comment:
Please, add named constants for 512 and 131072
And (nitpicking) 131072 looks more human-readable-ish when written like `128 << 10`